docs/_future/deploy_your_app.md

```diff
@@ -0,0 +1,14 @@
+---
+
+title: Deploy your app
+lang: en
+slug: deploy-your-app
+order: 6
+layout: future
+---
+
+<div class="section-content">
+Instructions for deploying your next-generation Bolt Python application to third-party infrastructure are coming soon! Stay tuned.
+</div>
+
+<p class="alert alert_info"><ts-icon class="ts_icon_info_circle"></ts-icon>Our next-generation platform is currently in beta. <a href="https://api.slack.com/future/survey" target="_blank">Your feedback is most welcome</a> - all feedback will help shape the future platform experience!</p>
```

Comment:
Promote the banner to right under the header?

Reply:
Yess this does feel better 🥇